### DISCUSSION TITLE
Server location, changing to singapore server. Is there an option?

### DISCUSSION THREAD

**Discussion by kenjiwb** [vercel/community#5646] [Help] [answered]
### Summary

Hi i would like to know if i were like it to be deployed on a server in singapore, does vercel provide this option? 

### Steps to Reproduce

Hi i would like to know if i were like it to be deployed on a server in singapore, does vercel provide this option?

**Comment by leerob** [comment, marked as answer]
You can configure your function region to be `sin1` for Singapore.

https://vercel.com/docs/functions/configuring-functions/region
